Libraries/PeriphDrivers/Include/MAX32655/tmr.h

```diff
@@ -154,6 +154,23 @@ typedef void (*mxc_tmr_complete_t)(int error);
  */
 int MXC_TMR_Init(mxc_tmr_regs_t *tmr, mxc_tmr_cfg_t *cfg, bool init_pins);
 
+/**
+ * @brief   Save the TMRn's Clock Source Frequency.
+ * @param   tmr          Pointer to timer module to save clock source frequency.
+ * @param   clksrc_freq  Clock Source Frequency (Hz) value to save.
+ */
+void MXC_TMR_SetClockSourceFreq(mxc_tmr_regs_t *tmr, int clksrc_freq);
```

Comment:
This should also not be a publicly accessible API function unless it's updated to actually set the clock source as well.  In which case it should become `MXC_TMR_SetClockSource`

If left as its current implementation move this inside of the C files.